MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -93,6 +97,7 @@ protected Texture2D(GraphicsDevice graphicsDevice, int width, int height, bool m
             this.GraphicsDevice = graphicsDevice;
             this.width = width;
             this.height = height;
+            this._texelSize = new Vector2(1f / (float)width, 1f / (float)height);
```

Comment:
It looks like we're relying on the graphics API to fail if the dimensions are zero, so to avoid a divide by zero we should validate width and height > 0 before using them in the Texture2D constructor.